.github/jobscripts/detect_dump.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -eux
+
+OUT=$(find ./ -name ".git*" -type d -prune -o -name "lib" -prune -o -name "var" -prune -o -name "bundles" -prune -o -name "cache" -type d -prune -o -name "ace" -type d -prune -o -type f -print0 | xargs -0 grep --color "dump(" | grep -v "Yaml::dump(") || true
```

Comment:
Can we make this more readable like this?

```
find ./ -name ".git*" -type d \
  -prune -o -name "lib" \
  -prune -o -name "var" \
  -prune -o -name "bundles" \
...
```

Reply:
I've added the newlines